packages/notifier/test/test-publish-kit.js

```diff
@@ -390,15 +346,15 @@ test('subscribeAfter bounds checking', async t => {
 
 test('subscribeAfter resolution sequencing', async t => {
   // Demonstrate sequencing by publishing to two destinations.
-  const { publisher: pub1, subscriber: sub1 } = makeEmptyPublishKit();
-  const { publisher: pub2, subscriber: sub2 } = makeEmptyPublishKit();
+  const { publisher: pub1, subscriber: sub1 } = makePublishKit();
+  const { publisher: pub2, subscriber: sub2 } = makePublishKit();
   const sub2LIFO = [];
 
   const sub1FirstAll = [];
-  Promise.resolve(sub1.subscribeAfter()).then(result =>
+  void Promise.resolve(sub1.subscribeAfter()).then(result =>
     sub1FirstAll.push(result),
   );
```

Comment:
Might as well be consistent here.
```suggestion
  Promise.resolve(sub1.subscribeAfter())
    .then(result => sub1FirstAll.push(result))
    .catch(t.fail);
```